app/controllers/photos_controller.rb

```diff
@@ -50,21 +52,29 @@ def update
 
   # DELETE /photos/1 or /photos/1.json
   def destroy
-    @photo.destroy
-    respond_to do |format|
-      format.html { redirect_back fallback_location: root_url, notice: "Photo was successfully destroyed." }
-      format.json { head :no_content }
-    end
+      @photo.destroy
```

Comment:
You'll most likely need to add an instance of the photo class in order to successfully destroy a photo.

Reply:
done?